Client/App.xaml.cs

```diff
@@ -54,10 +55,15 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             {
                 if (rootFrame.Content == null)
                 {
-                    // When the navigation stack isn't restored navigate to the first page,
-                    // configuring the new page by passing required information as a navigation
-                    // parameter
-                    rootFrame.Navigate(typeof(MainPage), e.Arguments);
+                    //Go to mainpage when clienID isnt null. When it is not null it means that the user has a session.
+                    if (ApplicationData.Current.LocalSettings.Values["clientId"] != null)
+                    {
+                        rootFrame.Navigate(typeof(MainPage), e.Arguments);
+                    }
+                    else
+                    {
+                        rootFrame.Navigate(typeof(LoginPage), e.Arguments);
+                    }
```

Comment:
We don't use sessions, it is also violating with the protocol. So this code isn't used. In my new branche the LoginPage is loaded by default as start page.